addons/api/mirage/config.js

```diff
@@ -81,7 +81,17 @@ export default function () {
     return authMethods.create(attrs);
   });
   this.get('/auth-methods/:id');
-  this.patch('/auth-methods/:id');
+  this.patch(
```

Comment:
Is this something we discovered during this work, or part of this refactor?

Reply:
It's the same issue as with the `POST` in #1491. I noticed when updating the auth method we would get a translation error because there's no `state`

Reply:
Ahh totally right!! Good job!